.changelog

```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+-  Added aarch64 support for selection of clients in the Installation
```

Comment:
Wrong changelog format, it seems. Please agree with the squad what format will be used for the changelogs, according to the changelog regulations I sent to the mailing lists.

If you won't bump versions for packages (as you do now for 4.1 and 4.2), then your format for this PR is OK, as the result will be:

```
- Change A
- Change B
- Change C
```
However this is not OK for Uyuni, where we bump versions, as the changelog should look like as:
```
- Version 2021.10
  * Change A
  * Change B
  * Change C
```
Please agree with the squad what to do. In the end if you don't manage this at the `.changelog`, whoever prepare the SRs will need to adjust the format.

Reply:
Changed. Thanks.